torch/_dynamo/variables/functions.py

```diff
@@ -1545,7 +1561,14 @@ def call_function(
             def wraps(fn):
                 if isinstance(fn, variables.NestedUserFunctionVariable):
                     return fn.clone(wrapped_fn=args[0])
-                unimplemented(f"functools.wraps({fn})")
+                unimplemented_v2(
+                    gb_type="functools.wraps",
+                    context=f"{fn}",
+                    explanation="`torch.compile` currently can't trace these",
```

Comment:
This seems more like we're not able to trace functools.wraps on functions defined outside the compile region.

Reply:
Yeah lemme refine the explanation. It's also a strange limitation from our end...